src/map/map.cpp

```diff
@@ -946,6 +946,9 @@ int32 map_config_default()
     map_config.craft_day_matters = 1;
     map_config.craft_moonphase_matters = 0;
     map_config.craft_direction_matters = 0;
+    map_config.craft_success_chance_multiplier = 1.0f;
```

Comment:
Yes. The already existing craft_chance_multiplier affects the chance of getting a skill up on a craft when a skill up is possible. My added craft_success_chance_multiplier affects the chance of successfully crafting. I can see how this is confusing, but I did try to name my modifier as close to what it does as possible.